schemas/system/Schema.schema

```diff
@@ -1,6 +1,6 @@
 System({
-  Category: { category: 'Category', display: true, required: true, index: true },
-  Version: { domain: 'Version', display: true, required: true, index: true },
+  Category: { category: 'Category', lookup: true, required: true, index: true },
+  Version: { domain: 'Version', required: true, index: true },
```

Comment:
So now we cannot get a specific version of a Schema?

Reply:
We can select it but now we will limit to single `lookup` fields per category just for simplicity. This will affect just lookups: comboboxes, gui lists, lookup pop-upp etc. In this certain case we will never select more than 1 version of schema is a single query so Category field will be unique for lookup.